pages/pgql-2.0-spec.md

```diff
@@ -4580,7 +4639,7 @@ All inserted entities must be identified with a variable name that has to be uni
 So the following query should fail, because the variable `x` is not only local to the vertex insertion term:
```

Comment:
I'm trying to understand what exactly this sentence means and how the next query is an example of this.
The query below fails because of the duplicate variable, right? (and because we didn't specify a label)

Reply:
that's also what I understood of it. But let me just remove this example because I believe in the rest of this spec we never give examples of queries that don't work. Otherwise we would have to add non-working examples everywhere.